src/main/java/com/flightstats/hub/dao/TtlEnforcer.java

```diff
@@ -2,38 +2,47 @@
 
 import com.flightstats.hub.model.ChannelConfig;
 import com.flightstats.hub.util.Commander;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import lombok.extern.slf4j.Slf4j;
 
+import javax.inject.Inject;
 import java.io.File;
-import java.util.Arrays;
-import java.util.HashSet;
+import java.util.Collection;
+import java.util.Optional;
 import java.util.Set;
 import java.util.function.Consumer;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
+@Slf4j
 public class TtlEnforcer {
-    private final static Logger logger = LoggerFactory.getLogger(TtlEnforcer.class);
 
-    public static void enforce(String path, ChannelService channelService,
-                               Consumer<ChannelConfig> channelConsumer) {
+    private static final String LOST_AND_FOUND_DIR = "lost+found";
+    private final Commander commander;
+
+    @Inject
+    public TtlEnforcer(Commander commander) {
+        this.commander = commander;
+    }
+
+    public void enforce(String path, ChannelService channelService,
+                        Consumer<ChannelConfig> channelConsumer) {
```

Comment:
While we're in here, can we rename this to something more specific? We pass methods called `handleCleanup` into both usages of this method, so that might inform some better name (`itemCleanupHandler`?). I haven't looked closer to see if there's anything in those consumers that's common to both...

Reply:
I only mention looking for commonalities, because having this generic consumer (that's supposed to do the bulk of the enforcing but is kind of a black box) is a little strange. I wonder if we could make it a function that returns files to be deleted or something.

Reply:
...that sort of decoupling of finding and removing files might have performance implications. Hm. Nevermind.